test_plans/runtime_classes.asciidoc

```diff
@@ -0,0 +1,52 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for context class and device class
+
+This is a test plan for the context class as described in Section 4.6.3. of the SYCL 2020 specification and for the device class as described in Section 4.6.4. of the SYCL 2020 specification. This test plan adds coverage for the missing functionality that is part of SYCL 2020.
+
+Estimated development time is one day.
+
+== Testing scope
+
+=== Backend coverage
+
+All the tests described below are not backend-specific and are performed for any SYCL backend.
+
+=== Device coverage
+
+All tests construct a test device for which conformance is assessed. All the tests described below are performed once for that test device.
+
+== Tests
+No negative test are included.
+
+=== Context
+
+==== Existing test
+The existing test in `context/context_info.cpp` checks the return value of `sycl::context.get_info` using `check_get_info_param` for various context information descriptors.
```

Comment:
```suggestion
The existing test in `context/context_info.cpp` checks the return value of `sycl::context::get_info` using `check_get_info_param` for various context information descriptors.
```

Reply:
Fixed, also for `sycl::device.get_info`.